packages/apps/src/Apps.tsx

```diff
@@ -17,12 +17,15 @@ import { SideBarTransition, SIDEBAR_MENU_THRESHOLD } from './constants';
 import Content from './Content';
 import SideBar from './SideBar';
 import BN from 'bn.js';
+import routing from '@polkadot/apps-routing';
+import { useLocation } from 'react-router-dom';
 
 interface SidebarState {
   isCollapsed: boolean;
   isMenu: boolean;
   isMenuOpen: boolean;
   transition: SideBarTransition;
+  isAdvance: boolean;
```

Comment:
Can you plz give more details on this constant, and how would it affect the landing page?

Reply:
Yes.. I will add it.. This is when 
<img width="322" alt="Screen Shot 2020-03-24 at 2 23 53 PM" src="https://user-images.githubusercontent.com/29415595/77378554-1ffa8a80-6ddb-11ea-9747-7b59943393f8.png"> open advanced is clicked, we need to pass this information to have the advanced dropdown open..